include/alpaka/math/MathStdLib.hpp

```diff
@@ -65,6 +68,9 @@ namespace alpaka
             , public SqrtStdLib
             , public TanStdLib
             , public TruncStdLib
+            , public IsnanStdLib
```

Comment:
Please provide the first commit with new math function within a separate PR.
This improvement is very useful but sneaked in via an independent RNG PR. It is not possible to write a changelog if this commit stays in this PR.

Reply:
https://github.com/alpaka-group/alpaka/pull/1446